src/Components/GEOSldas_GridComp/GEOSmetforce_GridComp/LDAS_Forcing.F90

```diff
@@ -1898,7 +1928,295 @@ subroutine get_conus_netcdf(date_time, met_path, N_catd, tile_coord, &
+
+    ! local variables
+
+    real,    dimension(era5_grid_N_lon,era5_grid_N_lat) :: tmp_grid, era5_lat2D, era5_lon2D
+    integer, dimension(N_era5_compressed)               :: land_i_era5, land_j_era5, p2g
+    real,    dimension(N_era5_compressed)               :: era5_lon, era5_lat
+    integer, dimension(N_catd)                          :: i_ind, j_ind
+    real,    dimension(N_era5_compressed)               :: tmp_vec
+    real,    dimension(N_catd,N_era5_vars)              :: force_array
+    integer, dimension(2)                               :: start, icount
+
+    integer :: k, n, hours_in_month, era5_var, ierr, ncid, era5_varid, kk
+    real    :: tol,  this_lon, this_lat, geos2era_tile_SQdistance
+
+    character(4)                :: YYYY
+    character(2)                :: MM
+    character(300)              :: fname
+    character(len=*), parameter :: Iam = 'get_ERA5_LIS'
+    character(len=400)          :: err_msg
+
+    ! era5_SQdistance is a half diagonal of 0.25deg grid box squared
+    real, parameter             :: era5_SQdistance = 0.03125  ![degrees] (0.125**2+0.125**2)
+
+    ! ----------------------------------------------------------------------------------------
+    
+    nodata_forcing = nodata_era5
+
+    tol = abs(nodata_forcing*nodata_tolfrac_generic)
+
+    ! assemble year and month strings
+    write (YYYY,'(i4.4)') date_time%year
+    write (MM,  '(i2.2)') date_time%month
+
+    ! compressed space dimension (always read global vector)
+    
+    start(1)  = 1
+    icount(1) = N_era5_compressed
+
+    ! time dimension (first entry in ERA5_LIS file is at 00Z)
+
+    if ( (date_time%min/=0) .or. (date_time%sec/=0) .or.           &
+         (mod(date_time%hour,dt_era5_in_hours)/=0)        ) then
+
+       call ldas_abort(LDAS_GENERIC_ERROR, Iam, 'timing ERROR!!')
+
+    end if
+
+    hours_in_month = (date_time%day-1)*24 + date_time%hour
+
+    start(2)  = hours_in_month / dt_era5_in_hours + 1
+    icount(2) = 1
+
+    ! compute indices for the nearest neighbor interpolation from ERA5 grid 
+    ! to tile space
+
+    do k=1,N_catd
+       
+       ! ll_lon and ll_lat refer to lower left corner of grid cell
+       ! (as opposed to the grid point in the center of the grid cell)
+       
+       this_lon = tile_coord(k)%com_lon
+       this_lat = tile_coord(k)%com_lat
+       
+       ! add small offsets to avoid unpredictable assignment of
+       ! regularly spaced tiles (such as from the EASEv2 tile space)
+       ! to ERA5 grid cells along certain lat/lon values
+       ! (that is, make it easier to reproduce the mapping done here
+       !  in separate postprocessing analysis scripts)
+       
+       this_lon = this_lon + 0.0001
+       this_lat = this_lat + 0.0001
+       
+       i_ind(k) = ceiling( (this_lon - era5_grid_ll_lon)/era5_grid_dlon )
+       j_ind(k) = ceiling( (this_lat - era5_grid_ll_lat)/era5_grid_dlat )
+
+       ! NOTE: For a "date line on center" grid and (180-dlon/2) < lon < 180 
```

Comment:
The ERA5_LIS grid is **not** a date-line-on-center grid.  Is this actually needed here?  I wouldn't think so.

Reply:
Now I see the problem.  Use function NINT  instead of ceiling can solve the problem

Reply:
> The ERA5_LIS grid is **not** a date-line-on-center grid. Is this actually needed here? I wouldn't think so.
Probably not needed, but I need to take time to do a deeper investigation. Initially, I concluded this should not cause a problem if stays and never came back to revisit (will do).

Reply:
I took a closer look, ran a couple of tests, confirmed the i_ind never exceeds the "era5_grid_N_lon" value. This part can be removed.